about.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="about.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet" />
+    <title>About page</title>
+  </head>
+  <body>
+    <header>
+      <div class="mobile no-pop" id="mobile-header">
+        <a id='mobile-menu' href="#">
+          <img id="hamburger-menu" src="./assets/mobile_menu.png" alt="hamburger-menu icon" />
+        </a>
+      </div>
+      <div class="desktop" id="top-bar">
+        <a href="#">
+          <img src="./assets/facebook.png" alt="facebook">
+        </a>
+        <a href="#">
+          <img src="./assets/twitter.png" alt="twitter">
+        </a>
+        <a href="#">English</a>
+        <a href="#">My page</a>
+        <a href="#">Logout</a>
+      </div>
+      <nav class="desktop" id="nav">
+        <div id="logo">
+          <a href="./index.html"><img src="./assets/NATO-Logo(4).png" alt="NATO logo"></a>
+        </div>
+        <div id="nav-links">
+          <ul>
+            <li>
+              <a id="nav-link-1" class="regular-link" href="./about.html">About</a>
```

Comment:
- [ ]  You did great work @znation2010   😃, however, in this `about.html` page we are not supposed to have again the `about` button in your navbar. What do you think if we change it to this `<a id="nav-link-1" class="regular-link" href="./index.html">Home</a>` to facilitate the user to go back to the `home` page 👍. Kindly fix it and make sure it is the same for the `mobile menu` 💪.  


- [ ] Also your buttons are taking the same color as a hover button. Kindly fix it 👍 

![Screenshot 2023-05-31 121439](https://github.com/znation2010/capstone-project-Conference-page/assets/57408419/b0a44061-66fe-4066-9ded-6851ee36e45e)


- [ ] Kindly, make sure, when we are at the desktop version the nav suppose to display none automatically 👍 . let's make it responsive 💪 

![Screenshot 2023-05-31 121936](https://github.com/znation2010/capstone-project-Conference-page/assets/57408419/70e6b0d3-dd43-468e-bdb0-967a3234d9e6)

Reply:
hi brother can you tell what to do to make  the desktop version the nav suppose to display none automatically 👍 . let's make it responsive

Reply:
Hi @znation2010, as you worked on responsiveness, you may just detect the screen and set a value to display the nav 💪